system/HTTP/Response.php

```diff
@@ -817,6 +830,7 @@ public function redirect(string $uri, string $method = 'auto', int $code = null)
 	 * @param string       $prefix   Cookie name prefix
 	 * @param boolean      $secure   Whether to only transfer cookies via SSL
 	 * @param boolean      $httponly Whether only make the cookie accessible via HTTP (no javascript)
+	 * @param null         $samesite
```

Comment:
need include `string` in `@param`:

```suggestion
       * @param string|null  $samesite
```